src/SFML/Audio/SoundFileReaderWav.cpp

```diff
@@ -307,6 +307,63 @@ bool SoundFileReaderWav::parseHeader(Info& info)
                 if (!decode(*m_stream, channelMask))
                     return false;
 
+                // NOLINTBEGIN(readability-identifier-naming)
+                // For WAVE channel mapping refer to: https://learn.microsoft.com/en-us/previous-versions/windows/hardware/design/dn653308(v=vs.85)#default-channel-ordering
+                static constexpr auto SPEAKER_FRONT_LEFT            = 0x1u;
+                static constexpr auto SPEAKER_FRONT_RIGHT           = 0x2u;
+                static constexpr auto SPEAKER_FRONT_CENTER          = 0x4u;
+                static constexpr auto SPEAKER_LOW_FREQUENCY         = 0x8u;
+                static constexpr auto SPEAKER_BACK_LEFT             = 0x10u;
+                static constexpr auto SPEAKER_BACK_RIGHT            = 0x20u;
+                static constexpr auto SPEAKER_FRONT_LEFT_OF_CENTER  = 0x40u;
+                static constexpr auto SPEAKER_FRONT_RIGHT_OF_CENTER = 0x80u;
+                static constexpr auto SPEAKER_BACK_CENTER           = 0x100u;
+                static constexpr auto SPEAKER_SIDE_LEFT             = 0x200u;
+                static constexpr auto SPEAKER_SIDE_RIGHT            = 0x400u;
+                static constexpr auto SPEAKER_TOP_CENTER            = 0x800u;
+                static constexpr auto SPEAKER_TOP_FRONT_LEFT        = 0x1000u;
+                static constexpr auto SPEAKER_TOP_FRONT_CENTER      = 0x2000u;
+                static constexpr auto SPEAKER_TOP_FRONT_RIGHT       = 0x4000u;
+                static constexpr auto SPEAKER_TOP_BACK_LEFT         = 0x8000u;
+                static constexpr auto SPEAKER_TOP_BACK_CENTER       = 0x10000u;
+                static constexpr auto SPEAKER_TOP_BACK_RIGHT        = 0x20000u;
+                // NOLINTEND(readability-identifier-naming)
+
+                info.channelMap.clear();
+
+                auto checkChannel = [&](auto bit, auto soundChannel)
```

Comment:
I didn't see in the code style guideline if we always const variables when possible but if so maybe consider a const here.